README.md

```diff
@@ -1 +1,11 @@
-# Theia - GitHub Extension
+# Theia - GitHub Plug-in
+
+The GitHub plug-in allows to generate and upload public SSH key to GitHub.
+
+Adds the following commands (shortcut `F1`):
+- GitHub: Authenticate ...
+- GitHub: Generate Ssh key and upload it to GitHub ...
+
```

Comment:
So this is it ?
What does it mean for the end user ? any `git` commands from the terminal or Theia UI would use the ssh key automatically if it is a ssh remote ?

Reply:
The issue is about reworking existing functionality to plug-in API. The extension (master) doesn't have any functionality except github Api. So That's even more then needed. I agree that we can have more functions, but that's another issues.

Reply:
OK do we have any GH-issue to implement the basic features ? can we plan it ?
Generate ssh key, uploading it and ... not being able to use it afterward ... Finally, this plugin/extension is useless.

Reply:
> OK do we have any GH-issue to implement the basic features ? can we plan it ?

We have such issue: https://github.com/eclipse/che/issues/8741 and we have a PR for extension version. I think we need to open a new PR after merging this.
> Generate ssh key, uploading it and ... not being able to use it afterward ... Finally, this plugin/extension is useless.

Existing functionality fully fits to requirements of the related issue. We can update the plugin later.